docs/source/release_notes.rst

```diff
@@ -3,14 +3,21 @@
 Release Notes
 -------------
 
-Future Release
-==============
+.. Future Release
+    ==============
     * Enhancements
     * Fixes
     * Changes
-        * Remove python-dateutil dependency requirement (:pr:`48`)
     * Documentation Changes
     * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v2.0.1 Apr 25, 2022
+==============
```

Comment:
Super minor detail at the moment but if we ever generate docs for this repo I think we need the `=` underline to be the same length of the text to avoid error.

Reply:
fixed